### ISSUE TITLE
Dragable on Ipad

### ISSUE THREAD

**Issue by robertaonnis** [RWAP/jquery-ui-touch-punch#28]
Hi,
on Ipad pro, if I use Dragable or Sortable the system move correcly  move the part but the is a problem on the scroll of the page.
If you try to move on touchscreen in the same time the page scroll and is really difficult to move.

**Comment by robertaonnis** [comment]
Also i can tell to you the sortable work on ipad pro but dragable I really don't know how move.
You also can test setting ipad pro on Chrome browser

**Comment by RWAP** [comment]
Yes - unfortunately, that is probably more to do with the touchscreen operating system and how it detects that you are trying to scroll the page, rather than move an item about the page - after all, scroll is probably much more common!

**Comment by robertaonnis** [comment]
Ok so this bug remain. Have you plan to check in the future or not yet?
Thanks a lot

**Comment by RWAP** [comment]
I don't own an ipad or any iOS device - so cannot test.  Open to ideas as to how the operating system is supposed to know that you are dragging something beyond the edge of the screen and so you want to continue scrolling on the page as well as dragging.....

**Comment by RWAP** [comment]
Just read somewhere that iOS expects:

"You single-finger click to start dragging something over to wherever you want it to go; then while holding that finger, an additional two finger gesture can be used to scroll content in any application."

Does that work - or is it what you are trying to achieve?

**Comment by robertaonnis** [comment]
I know how you can test so you can see the issue on console.
Open chrome and set responsive like ipad air:

![image](https://user-images.githubusercontent.com/73394579/209864087-af0e656b-ecbe-4573-bd03-9312d3839f50.png)

This port view is the same of ipad pro. And the system replicate exactly the issue

**Comment by RWAP** [comment]
Yes, I know this, but with a mouse, you cannot replicate two finger movements etc...

Do you have a sample site where you have implemented the draggable option......

People also need to be aware that this is NOT my code - it is just a repository where we brought together all of the requested changes to the original code which had not been updated in years.  

Others can comment and suggest their own changes, if they can spot what is causing an issue, or have a workaround

**Comment by RWAP** [comment]
> I know how you can test so you can see the issue on console. Open chrome and set responsive like ipad air:
> 
> ![image](https://user-images.githubusercontent.com/73394579/209864087-af0e656b-ecbe-4573-bd03-9312d3839f50.png)
> 
> This port view is the same of ipad pro. And the system replicate exactly the issue

Someone else has reported that this works in mobile Chrome, but not Firefox.

If you need it investigating, then you need an example site and confirm which browser you are using!
